src/lib.js

```diff
@@ -0,0 +1,217 @@
+import loadHandlerManager from './stateManagement/loadHandlerManager.js';
+import {
+  newImageIdSpecificToolStateManager,
+  globalImageIdSpecificToolStateManager
+} from './stateManagement/imageIdSpecificStateManager.js';
+import {
+  newFrameOfReferenceSpecificToolStateManager,
+  globalFrameOfReferenceSpecificToolStateManager
+} from './stateManagement/frameOfReferenceStateManager.js';
+import {
+  setToolPassiveForElement,
+  setToolActiveForElement,
+  setToolEnabledForElement,
+  setToolDisabledForElement
+} from './store/setToolMode.js';
+import {
+  state,
+  getters,
+  modules
+} from './store/index.js';
+import {
+  getNewContext,
+  draw,
+  path,
+  setShadow,
+  drawLine,
+  drawLines,
+  drawJoinedLines,
+  drawCircle,
+  drawEllipse,
+  drawRect,
+  fillOutsideRect,
+  fillBox,
+  fillTextLines
+} from './util/drawing.js';
+import drawTextBox from './util/drawTextBox.js';
+import drawArrow from './util/drawArrow.js';
+import getLuminance from './util/getLuminance.js';
+import copyPoints from './util/copyPoints.js';
+import calculateSUV from './util/calculateSUV.js';
+import calculateEllipseStatistics from './util/calculateEllipseStatistics.js';
+import setContextToDisplayFontSize from './util/setContextToDisplayFontSize.js';
+import scrollToIndex from './util/scrollToIndex.js';
+import scroll from './util/scroll.js';
+import roundToDecimal from './util/roundToDecimal.js';
+import {
+  projectPatientPointToImagePlane,
+  imagePointToPatientPoint,
+  planePlaneIntersection
+} from './util/pointProjector.js';
+
+import pointInsideBoundingBox from './util/pointInsideBoundingBox.js';
+import pointInEllipse from './util/pointInEllipse.js';
+import makeUnselectable from './util/makeUnselectable.js';
+import getRGBPixels from './util/getRGBPixels.js';
+import {
+  getDefaultSimultaneousRequests,
+  getMaxSimultaneousRequests,
+  getBrowserInfo,
+  isMobileDevice
+} from './util/getMaxSimultaneousRequests.js';
+import wwwcSynchronizer from './synchronization/wwwcSynchronizer.js';
+import updateImageSynchronizer from './synchronization/updateImageSynchronizer.js';
+import Synchronizer from './synchronization/Synchronizer.js';
+import stackScrollSynchronizer from './synchronization/stackScrollSynchronizer.js';
+import stackImagePositionSynchronizer from './synchronization/stackImagePositionSynchronizer.js';
+import stackImagePositionOffsetSynchronizer from './synchronization/stackImagePositionOffsetSynchronizer.js';
+import stackImageIndexSynchronizer from './synchronization/stackImageIndexSynchronizer.js';
+import panZoomSynchronizer from './synchronization/panZoomSynchronizer.js';
+import requestPoolManager from './requestPool/requestPoolManager.js';
+
+export const lib = {
+  store: {
+    state,
+    getters,
+    modules,
+    setToolMode: {
+      setToolPassiveForElement,
+      setToolActiveForElement,
+      setToolDisabledForElement,
+      setToolEnabledForElement
+    }
+  },
+  core: {
```

Comment:
I go back and forth on this. I'm not sure if we need to qualify what is an is not core.